--- a/lib/src/audio/widgets/ayah_audio_widget.dart
+++ b/lib/src/audio/widgets/ayah_audio_widget.dart
@@ -79,7 +79,7 @@
                         ),
                         AnimatedSize(
                           duration: const Duration(milliseconds: 300),
-                          alignment: AlignmentGeometry.bottomCenter,
+                          alignment: Alignment.bottomCenter,
                           child: AnimatedCrossFade(
                             // مطابقة مدة الأنيميشن مع السلايدر لتجنب overflow
                             // Match animation duration with slider to avoid overflow

